flask/test/auth_tests/auth.py

```diff
@@ -19,33 +19,46 @@ def wrapper(*args):
 
     return wrapper
 
-
 def get_user_token():
     """Grabs a user token for Auth0
     Data structure as described here
     https://manage.auth0.com/dashboard/eu/boxtribute-dev/apis/5ef3760527b0da00215e6209/test"""  # line too long # noqa: E501
 
     auth0_domain = os.getenv("AUTH0_DOMAIN")
-    auth0_client_id = os.getenv("AUTH0_CLIENT_ID")
+    auth0_client_id = os.getenv("AUTH0_CLIENT_TEST_ID")
     auth0_audience = os.getenv("AUTH0_AUDIENCE")
-    auth0_secret = os.getenv("AUTH0_CLIENT_SECRET")
+    auth0_secret = os.getenv("AUTH0_CLIENT_SECRET_TEST")
     auth0_username = os.getenv("AUTH0_USERNAME")
     auth0_password = os.getenv("AUTH0_PASSWORD")
 
+    print(auth0_domain)
+    print(auth0_client_id)
+    print(auth0_secret)
+    print(auth0_audience)
+    print(auth0_username)
+    print(auth0_password)
+
```

Comment:
Instead of printing line-by-line you could `print(parameters)` in line 50 if it's not solely all for debugging anyways?